store.go

```diff
@@ -2705,7 +2764,7 @@ func (s *store) Changes(from, to string) ([]archive.Change, error) {
 }
 
 func (s *store) DiffSize(from, to string) (int64, error) {
-	lstore, err := s.LayerStore()
+	lstore, err := s.ROLayerStore()
```

Comment:
Pretty sure this needs to be a writeable store, in order for `Changes()` to be able to do its work reliably.  Attempts to compute this information on read-only stores have a good chance of erroring out on read-only filesystems.

Reply:
Alright, changed it to `RWLayerStore()`